test_collections/matter/sdk_tests/support/python_testing/sdk_python_tests.py

```diff
@@ -170,26 +177,32 @@ def sdk_mandatory_python_test_collection(
 
 def custom_python_test_collection(
     python_test_folder: SDKTestFolder = CUSTOM_PYTHON_TEST_FOLDER,
+    tests_file_path: Path = CUSTOM_PYTHON_TESTS_PARSED_FILE,
 ) -> Optional[PythonCollectionDeclaration]:
+    asyncio.run(
+        generate_python_test_json_file(
+            test_folder=CUSTOM_PYTHON_SCRIPTS_FOLDER,
+            json_output_file=COMPLETE_JSON_OUTPUT_FILE_FOLDER
+            / "custom_python_tests_info.json",
```

Comment:
Please correct me if I'm wrong, but it seems like `COMPLETE_JSON_OUTPUT_FILE_FOLDER` is the same as `SDK_TESTS_ROOT`.
In that case, `COMPLETE_JSON_OUTPUT_FILE_FOLDER / "custom_python_tests_info.json"` is the same as `CUSTOM_PYTHON_TESTS_PARSED_FILE`.
If that's true, then I suggest using `CUSTOM_PYTHON_TESTS_PARSED_FILE` directly.

Reply:
Also, I don't understand why a constant that starts with "COMPLETE" actually always needs to be used with a complement. Maybe renaming it to just `JSON_OUTPUT_FILE_FOLDER` would be better?

Reply:
Good catch!